unit-tests/src/test/java/org/eclipse/collections/impl/map/fixed/TripletonMapTest.java

```diff
@@ -360,7 +361,18 @@ public void testEqualsAndHashCode()
     public void testClone()
     {
         MutableMap<Integer, String> map = new TripletonMap<>(1, "One", 2, "Two", 3, "Three");
-        Verify.assertShallowClone(map);
+        try
+        {
+            Verify.assertShallowClone(map);
+        }
+        catch (Exception e)
+        {
+            // Suppress if a Java 9 specific exception related to reflection is thrown.
```

Comment:
That would be an overkill because there is a check for Java 9 specific exception being thrown. Any regression breaks should fail the test regardless of Java version.